cordova-lib/src/cordova/plugin.js

```diff
@@ -410,6 +422,9 @@ function getPluginVariables(variables){
 }
 
 function getVersionFromConfigFile(plugin, cfg){
+    if (cordova_util.isScopedRegistryPluginSpec(plugin)) {
```

Comment:
Should we chack for this before calling `extractPluginId`? From what i understand, `extractPluginId` will return id without scope even if plugin spec contains scope expression.